openrtb_ext/imp_sharethrough.go

```diff
@@ -1,13 +1,18 @@
 package openrtb_ext
 
-type ExtImpSharethrough struct {
-	Pkey       string  `json:"pkey"`
-	Iframe     bool    `json:"iframe"`
-	IframeSize []int   `json:"iframeSize"`
-	BidFloor   float64 `json:"bidfloor"`
+type ExtData struct {
+	PBAdSlot string `json:"pbadslot"`
 }
 
 // ExtImpSharethrough defines the contract for bidrequest.imp[i].ext.sharethrough
+type ExtImpSharethrough struct {
+	Pkey       string   `json:"pkey"`
+	Iframe     bool     `json:"iframe"`
+	IframeSize []int    `json:"iframeSize"`
+	BidFloor   float64  `json:"bidfloor"`
+	Data       *ExtData `json:"data,omitempty"`
```

Comment:
You defined a new bidder parameter. Can you please update `static/bidder-params/sharethrough.json` file as well?

Reply:
@VeronikaSolovei9 do we have to? None of the other adapters which implement some sort of `ExtData` on their bidder parameters have declared it there. What is the correct way to do this?

Reply:
You don't have to but it is definitely preferred/recommended. You'll get parameter validation by PBS core upstream so you can be sure you have valid data in your ext when your adapter code executes.

Bidder param documentation can be found here: https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html#bidder-parameters

Reply:
@epechuzal I discussed this with the team and this is actually required because other systems and front-ends may use the defined schema.